src/main/java/seedu/recipe/storage/JsonSerializableRecipeBook.java

```diff
@@ -0,0 +1,60 @@
+package seedu.recipe.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.RecipeBook;
+import seedu.recipe.model.ReadOnlyRecipeBook;
+import seedu.recipe.model.recipe.Recipe;
+
+/**
+ * An Immutable RecipeBook that is serializable to JSON format.
+ */
+@JsonRootName(value = "addressbook")
```

Comment:
recipebook but maybe not if json still reads addressbook

Reply:
Changes will be done on my end in model.UserPrefs - approved